br/pkg/streamhelper/config/advancer_conf.go

```diff
@@ -9,17 +9,22 @@ import (
 )
 
 const (
-	flagBackoffTime      = "backoff-time"
-	flagTickInterval     = "tick-interval"
-	flagFullScanDiffTick = "full-scan-tick"
-	flagAdvancingByCache = "advancing-by-cache"
+	flagBackoffTime          = "backoff-time"
+	flagMaxBackoffTime       = "max-backoff-time"
+	flagTickInterval         = "tick-interval"
+	flagFullScanDiffTick     = "full-scan-tick"
+	flagAdvancingByCache     = "advancing-by-cache"
+	flagConsistencyCheckTick = "consistency-check-tick"
```

Comment:
maybe we should check these config name with @IANTHEREAL

Reply:
Those configs aren't exported. 🤔